shared/fs/folder/container.js

```diff
@@ -9,6 +9,8 @@ import * as FsGen from '../../actions/fs-gen'
 const mapStateToProps = (state, {path}) => ({
   _pathItem: state.fs.pathItems.get(path, Constants.unknownPathItem),
   _username: state.config.username,
+  // TODO: figure out how to get this info
+  offline: false,
```

Comment:
This is in `songgao/KBFS-4019-KBFS-4020` too, but I guess we can also connect this in a separate PR after that one's merged.

Reply:
I'd like to connect it but I don't know where to start.

Reply:
If you rebase this over `songgao/KBFS-4019-KBFS-4020`, this could probably be `kbfsDaemonStatus: state.fs.kbfsDaemonStatus` and then in mergeProps, you can combine this with the prefetchStatus from stateProps._pathItem.

Reply:
Oh just realized you edited the comment. Sounds good!